README.md

```diff
@@ -1,13 +1,19 @@
 # wordcel
```

Comment:
Unrelated to the PR, but this should be `Wordcel`

Reply:
done